kbd-mode.el

```diff
@@ -0,0 +1,245 @@
+;;; kbd-mode.el --- Font locking for kmonad's .kbd files -*- lexical-binding: t -*-
```

Comment:
Please don't do this.  If guix can't build it then that seems like an error in the build script more than an error in where the relevant elisp files are.

@UristMcKorobochka are you still interested in this?  As I said, if the style for `defvar`s declarations is really standardized then we may as well change to that.  If you could remove the other changes, rebase and squash your commits we can merge this!

Reply:
@slotThe 
I'll do that today (thx for the opportunity to say that)

#that seems like an error in the build script more than an error in where the relevant elisp files are.

@slotThe, looking in sources of packages defined here I see it as convention, the-way-things-work to have in package repo:
1) .el(s) in highest directory
or/and
2) makefile